app/tests/spec/views/complete_reset_password.js

```diff
@@ -361,9 +364,12 @@ function (chai, sinon, p, AuthErrors, Metrics, FxaClient, InterTabChannel,
                   EMAIL,
                   PASSWORD,
                   relier,
+                  // TODO: remove once https://github.com/eslint/eslint/issues/1801 gets fixed
+                  /* eslint-disable */
```

Comment:
Its a multiline statement, which is listed on one of the related issues. It can be fixed by de-indenting  the braces, making it look extremely ugly. @zaach